### ISSUE TITLE
Tabs under Tabs 

### ISSUE THREAD

**Issue by britt6612** [grommet/hpe-design-system#2530]
Needed guidance on how many levels of Tabs are ok? 
Need guidance for use case where there are two levels of tabs. 
<img width="477" alt="Screen Shot 2022-04-13 at 10 50 32 AM" src="https://user-images.githubusercontent.com/42451602/163230790-c4994bca-c959-4d01-b9f0-f62a4197d01f.png">

**Comment by vavalos5** [comment]
I don't think this is something we want to support. Two tables can be stacked to show different information. Should we close?
